src/ReadTrainer/PlayText.js

```diff
@@ -11,71 +11,132 @@ import {
   FontIcon
 } from "react-md";
 
+import { HotKeys } from "react-hotkeys";
+
 import { makeLogger } from "../analytics";
 
 import "./ReadTrainer.css";
 
+const keyMap = {
+  show: "space",
+  correct: "left",
+  incorrect: "right"
+};
+
 const event = makeLogger("ReadTrainer");
 
 const PlayHiddenCard = inject("store")(
-  observer(({ store, onShow }) => (
-    <Card className="bottom-card">
-      <CardTitle
-        title="Listen"
-        subtitle={store.morse.playing ? "Playing..." : "Waiting..."}
-      />
-      <CardText>
-        <p>Decode the text and press 'Show' when ready</p>
-      </CardText>
-      <CardActions centered>
-        <Button
-          raised
-          primary
-          onClick={onShow}
-          iconEl={<FontIcon>visibility</FontIcon>}
-        >
-          Show
-        </Button>
-      </CardActions>
-    </Card>
-  ))
+  observer(
+    class PlayHiddenCard extends Component {
+      componentDidMount() {
+        document.getElementById("PlayHiddenCard").focus();
+      }
+
+      render() {
+        return (
+          <HotKeys keyMap={keyMap}>
```

Comment:
Since this is needed both in PlayHiddenCard and PlayVisibleCard, would it make sense to instead have it in PlayText.render() ?

Reply:
I don't have a strong preference. The hotkeys functionality seems like a part of each card. However, moving the hotkeys functionality to the PlayText.render() would result in the body of PlayHiddenCard and PlayVisibleCard being less intended. So perhaps that it is a gain in readability?

I'll let you have the final say. Just let me know. :)

Reply:
Good point regarding the encapsulation -- I agree that the hotkeys belong to the card. Let's leave it as is.